src/directives/bind.js

```diff
@@ -36,7 +36,10 @@ export function handleAttributeBindingDirective(component, el, attrName, express
         } else if (el.tagName === 'SELECT') {
             updateSelect(el, value)
         } else {
+            // Cursor position should be restored back to origin due to a safari bug
+            const cursorPosition = el.selectionStart 
             el.value = value
+            el === document.activeElement ? el.setSelectionRange(cursorPosition, cursorPosition) : null
```

Comment:
I'm not sure a ternary should be used here since the else value is null. I'd instead use a regular if statement.

Reply:
Yeah, that's the right path.